src/ui/handler/dblclick_zoom.js

```diff
@@ -4,6 +4,10 @@ import { bindAll } from '../../util/util';
 
 import type Map from '../map';
 import type {MapMouseEvent, MapTouchEvent} from '../events';
+import Point from '@mapbox/point-geometry';
```

Comment:
`import type Point` if it's only used for typing.

Reply:
:+1: thanks for the catch! Fixed in https://github.com/mapbox/mapbox-gl-js/pull/8086/commits/ae80d83b06da759e149e560c667f58da6e1b9bf2